### ISSUE TITLE
[BITV] 9.3.3.2/2.4 - Modal dialog Emois (called up in "Online status" dialog) -  Search input field lacks visual label and there is no search button or symbol to help identify it as search field. (3)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/nextcloud-vue#3871]
see also https://github.com/nextcloud/server/issues/37091

- [x] https://github.com/nextcloud/nextcloud-vue/pull/3900
- [x] https://github.com/nextcloud/nextcloud-vue/pull/3953
- [x] https://github.com/nextcloud/server/pull/37586

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v2-n4

</details>

- [x] 9\.3.3.2/3.6 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v3-n6
- [x] 9\.3.3.2/10.6 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v10-n6

**Comment by AndyScherzinger** [comment]
is this the correct repo @nickvergessen ?
